.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
do we want this to be false? I don't see a great reason not to have it be false

Reply:
Yeah, I guess there's not a good reason -- if someone really believes one of the existing templates doesn't match their need, we can always revisit